packages-ts/gauntlet-cosmos-contracts/src/commands/contracts/ocr2/proposal/acceptProposal.ts

```diff
@@ -44,7 +45,7 @@ const validateRandomSecret: ValidateFn<CommandInput> = async (input) => {
 const validateOffchainConfig: ValidateFn<CommandInput> = async (input, context) => {
   const { offchainConfig } = await serializeOffchainConfig(
     input.offchainConfig,
-    process.env.SECRET!,
```

Comment:
get from the input rather than the env so it can be passed into flags if necessary